taskcluster/docker/node/build.py

```diff
@@ -63,6 +64,54 @@ def get_package_info():
     return contents
 
 
+def write_package_info(package_info):
+    with open("package.json", "w") as fh:
+        json.dump(package_info, fh)
+    assert get_package_info() == package_info
+
+
+def get_buildid():
+    now = datetime.utcnow()
+    return now.strftime("%Y%m%d%H%M%S")
+
+
+def get_buildid_version(version):
+    """Version schema check+append a `buildid{buildid}` to ensure unique version."""
+    # XXX is there a more precise schema we should verify ourselves against?
+    if len(version.split(".")) not in (1, 2, 3):
+        raise Exception("{version} has too many version parts!")
+    if "buildid" in version:
+        raise Exception("{version} already has a buildid specified!")
+    buildid_version = f"{version}buildid{get_buildid()}"
+    print(f"Buildid version is {buildid_version}")
+    return buildid_version
+
+
+def find_manifests():
+    manifest_list = []
+    for dir_name, subdir_list, file_list in os.walk("."):
+        for dir_ in subdir_list:
+            if dir_ in (".git", "node_modules"):
+                subdir_list.remove(dir_)
+                continue
+        if "manifest.json" in file_list:
+            manifest_list.append(f"{dir_name}/manifest.json")
+    return manifest_list
+
+
+def find_update_manifest_json(buildid_version):
+    """We also have manifest.json files; let's update them as well."""
+    for manifest in find_manifests():
+        print(f"Updating {manifest} version...")
```

Comment:
I'm updating the manifest.json version as well.
I think this is overall a win, in that we verify that the manifest.json and package.json versions match. Let me know if you have opinions here.